internal/cmd/plugin/command_install.go

```diff
@@ -0,0 +1,141 @@
+package plugin
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/go-yaml/yaml"
+	"github.com/hashicorp/go-version"
+	"github.com/spf13/cobra"
+
+	"github.com/confluentinc/cli/internal/pkg/errors"
+	"github.com/confluentinc/cli/internal/pkg/output"
+	"github.com/confluentinc/cli/internal/pkg/plugin"
+	"github.com/confluentinc/cli/internal/pkg/utils"
+)
+
+const (
+	programNotFoundMsg      = "[WARN] Unable to find %s. Check that it is installed in a directory in your $PATH.\n"
+	unableToParseVersionMsg = "[WARN] Unable to parse %s version.\n"
+	insufficientVersionMsg  = "[WARN] Installed %s version %s is less than the required version %s.\n"
+)
+
+func (c *command) newInstallCommand() *cobra.Command {
+	return &cobra.Command{
+		Use:   "install",
+		Short: "Install official Confluent CLI plugins.",
+		Long:  "Install official Confluent CLI plugins from the confluentinc/cli-plugins repository.",
+		Args:  cobra.ExactArgs(1),
+		RunE:  c.install,
+	}
+}
+
+func (c *command) install(cmd *cobra.Command, args []string) error {
+	home, err := os.UserHomeDir()
+	if err != nil {
+		return err
+	}
+	confluentDir := filepath.Join(home, ".confluent")
+	dir, err := os.MkdirTemp(confluentDir, "cli-plugins")
```

Comment:
Why are we using `MkdirTemp()` instead of `Mkdir()` here? What's the difference?

Reply:
Two reasons: it makes it clearer in the code that it's a temporary directory, and `MkdirTemp` doesn't require folder permissions as an argument.